tools/dragen-germline/4.3.6/dragen-germline__4.3.6.cwl

```diff
@@ -79,6 +79,22 @@ requirements:
             exit 1
           fi
 
+          # DRAGEN Multi-region Joint Detection (MRJD) is a de novo germline small variant caller for paralogous regions.
+          # MRJD is compatible with the hg38, hg19 and GRCh37 reference genomes.
+          # https://help.dragen.illumina.com/product-guides/dragen-v4.3/dragen-dna-pipeline/small-variant-calling/multi-region-joint-detection
+          #
+          # Multi Region Joint Detection (MRJD) Caller should be runs as standalone pipeline on DRAGEN™ server (not in integrated with Germline Small VC)
+          # https://support.illumina.com/content/dam/illumina-support/documents/downloads/software/dragen/release-notes/200056923_00_DRAGEN_4_3_6_Customer-Release-Notes.pdf
+          echo "run MRJD" 1>&2
+          eval /opt/edico/bin/dragen \\
```

Comment:
We also probably want the outputs to be in a subfolder of the output directory right, given this is another dragen invocation I'm worried running this will overwrite the existing outputs-replay.json + other files metrics generated by the standard snv germline dragen invocation

Reply:
If this is enabled, the mrjd output will be integrated with the small variants output. In this case, the current output definition should be fine?

Reply:
Ah so you want the mjrd dragen to run instead of the other dragen invocation?
If so, why is it written so that they run sequentially?

Reply:
From the release notes linked above

```
MRJD cannot run together with the Small Variant Caller in this version. The recommended 
usage is to run DNA Mapping and Small Variant Calling workflow first, and then run MRJD 
using the aligned BAM file generated from DNA Mapping workflow as input. 
```

Reply:
So this needs to be run **after** the small variant calling.  

> In this case, the current output definition should be fine? 

No, I'm not confident that the replay-json won't be overwritten.  The mrjd should be written to a subfolder, we can then also include the merged vcf in that subfolder as well